ci/workflows/pull-request.yml

```diff
@@ -17,6 +17,13 @@ steps:
           run: rw-build-env
           config: ci/docker-compose.yml
           mount-buildkite-agent: true
+      - telefonica/post#0.1.1:
+          post:
+            - when: failure
+              steps: |
+                - command: "cargo-make/makers logs"
```

Comment:
No need to run this command. If sqllogictest fails, logs are automatically stored in log folder. We simply need to upload them.